php/docker/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:18.04
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y --force-yes build-essential curl cmake g++ libpcre3-dev libssl-dev make openssl libgmp-dev git
+
+RUN apt-get install -y software-properties-common
+RUN LC_ALL=C.UTF-8 add-apt-repository -y ppa:ondrej/php
+RUN apt-get update
+RUN apt-get install -y php7.1 php7.1-dev php7.1-xml php7.1-mbstring php7.1-curl php7.1-zip php7.1-xdebug
+
+RUN apt-get install -y wget
+
+RUN cd /tmp && wget http://downloads.datastax.com/cpp-driver/ubuntu/16.04/dependencies/libuv/v1.24.0/libuv1_1.24.0-1_amd64.deb && wget http://downloads.datastax.com/cpp-driver/ubuntu/16.04/dependencies/libuv/v1.24.0/libuv1-dev_1.24.0-1_amd64.deb

```

Comment:
would be interesting to if this php driver works ontop of scylla cpp-driver fork
https://github.com/scylladb/cpp-driver/releases/tag/2.16.2b

and how hard it would be to fill the gap to make it happen

Reply:
it's quite sad that datastax didn't released version for ubuntu since 18.04...

Reply:
I know right? I saw a guy that implemented it in PHP8.1 but still waiting for his answer on how build it.

When I have a straight answer, I'll do a new pull request implementing a v2 of the project.

Reply:
I think the more interesting thing for us, is using scylladb fork, I don't know exactly how todo it, but we might be able to get the maintainer of scylladb cpp-driver to lend a hand

Reply:
Any help to figure out how to upgrade it to PHP 8.x will be awesome! Next week probably we can schedule something and see what is missing.